playground/infrastructure/helper.py

```diff
@@ -52,21 +58,94 @@ def find_examples(work_dir: str) -> List[Example]:
     Returns:
         List of Examples.
     """
-    examples = [Example("", "", SDK_UNSPECIFIED, "", "", "", STATUS_UNSPECIFIED)]
+    examples = []
+    for root, _, files in os.walk(work_dir):
+        for filename in files:
+            filepath = os.path.join(root, filename)
+            if _match_pattern(filepath):
+                examples.append(_get_example(filepath, filename))
```

Comment:
probably you need to add logic with checking the pattern as Ilya and Artur mentioned

Reply:
Changed.